src/Mutation.tsx

```diff
@@ -41,12 +42,13 @@ export declare type FetchResult<C = Record<string, any>, E = Record<string, any>
 export declare type MutationOptions<TData = any, TVariables = OperationVariables> = {
   variables?: TVariables;
   optimisticResponse?: Object;
-  refetchQueries?: string[] | PureQueryOptions[];
+  refetchQueries?: string[] | PureQueryOptions[] | RefetchQueriesProviderFn;
   update?: MutationUpdaterFn<TData>;
 };
 
 export interface MutationProps<TData = any, TVariables = OperationVariables> {
   mutation: DocumentNode;
+  ignoreResults?: boolean;
```

Comment:
Need to add a test in the `Mutation.test.tsx` file as a follow-up

Reply:
I added a simple test of this (could be expanded to more cases) on the next commit